setup.cfg

```diff
@@ -108,6 +108,7 @@ filterwarnings =
     ignore::DeprecationWarning:traittypes
     ignore::DeprecationWarning:voila
     ignore:::specutils.spectra.spectrum1d
+    ignore::DeprecationWarning:ipykernel
```

Comment:
This warning will go away once https://github.com/jupyter-widgets/ipywidgets/pull/3533 is merged and released upstream. FYI.

Reply:
I also committed 0877c09 directly on `main` for this since it is blocking other PRs. Please remove this change in `setup.cfg` and rebase. Thanks!